src/software/thunderscope/robot_diagnostics/robot_view.py

```diff
@@ -17,168 +15,44 @@ class RobotView(QWidget):
 
     """
 
-    toggle_robot_connection_signal = QtCore.pyqtSignal(int)
+    toggle_all_connection_signal = QtCore.pyqtSignal(int, int)
 
-    def __init__(self, load_diagnostics):
+    def __init__(self, load_fullsystem):
 
-        """Initialize the robot view."""
+        """
+        Initialize the robot view.
+        Sets up a Robot Info Widget for each robot
```

Comment:
Missing `load_fullsystem`. It should also have a comment explaining what setting this value to True/False does

Reply:
forgot about param docs for constructors, thanks!

Reply:
done